### PR TITLE
Implement Staff List design

### LINKED ISSUE #20: Apply "Staff" screen design
Blocked by #18 

## Overview (Required)
The title says everything.
Make it previewable(Prease add `@Preview` methods) and try to implement it paying attention to [state hoisting](https://developer.android.com/jetpack/compose/state#stateless-composables).
## Links
https://www.figma.com/file/IFlrbfmBSdYvUz7VmSzfLV/DroidKaigi_2021_official_app
There are some parts where the design size is not accurate. Use 4dp * x size and similar Material components.
Please log in to see the size on the design.

### PR DESCRIPTION
**Pull request by mkano9**
## Issue
- close #20 

## Overview (Required)
- Implement Staff List design according to https://www.figma.com/file/IFlrbfmBSdYvUz7VmSzfLV/DroidKaigi_2021_official_app?node-id=1%3A680
- Using `fakeStaffApi()` until the real API can return data.

### CODE REVIEW

File: uicomponent-compose/other/src/main/java/io/github/droidkaigi/feeder/staff/StaffItem.kt [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+            shape = CircleShape
+        ) {
+            NetworkImage(
+                url = staff.image,
+                modifier = Modifier.fillMaxSize(),
+                contentScale = ContentScale.Fit,
+                contentDescription = "Staff Icon"
+            )
+        }
+        Text(
```
Comment (mkano9) [inline uicomponent-compose/other/src/main/java/io/github/droidkaigi/feeder/staff/StaffItem.kt:61]: The design comp shows a name and a title for each staff, but the `Staff` model doesn't have the title text. `StaffApi` doesn't have it either, so I made it only display the name. I assume the design is outdated, but let me know if I'm wrong.

**Comment by tomoya0x00** [comment]
👀

File: uicomponent-compose/other/src/main/java/io/github/droidkaigi/feeder/staff/StaffItem.kt (hunk 1 above)
Comment (tomoya0x00) [reply uicomponent-compose/other/src/main/java/io/github/droidkaigi/feeder/staff/StaffItem.kt:61]: Thank you for handling this problem.
It's OK as it is!

**Review by tomoya0x00 — COMMENTED** [review]
@mkano9  Almost LGTM 👍
Could you fix ci warnings? You can just run `./gradlew :spotlessApply` to fix it.

**Comment by mkano9** [comment]
> @mkano9 Almost LGTM 👍
> Could you fix ci warnings? You can just run `./gradlew :spotlessApply` to fix it.

FYI, I keep getting errors with `:spotlessApply`. I just manually fixed my lint errors.

```
Could not determine the dependencies of task ':spotlessApply'.
> Could not create task ':spotlessKotlinApply'.
   > Could not create task ':spotlessKotlin'.
      > No such reference 'origin/main'
```

**Comment by tomoya0x00** [comment]
>  I just manually fixed my lint errors.

Thank you for your quick fix!

> FYI, I keep getting errors with :spotlessApply.

Oh ... I will check it. Thank you for your report 🙏

**Comment by mkano9** [comment]
@tomoya0x00 Great, thanks for your review 🚀

**Comment by takahirom** [comment]
Thank you 🙏

**Review by tomoya0x00 — APPROVED** [review]
Anyway, This PR looks good for me ❤️  Let's merge!
Thank you for your contribution 👍

**Pull request merged**